examples/javascript-sensors-viewer/README.md

```diff
@@ -0,0 +1,100 @@
+# Sensors Viewer example
+
+This is a sample web page made in vanilla Javascript that you can use to show available sensors of a
+device and their data.
+
+You can follow the walkthroughs below if you wish to try it out: you will first simulate a device
+that streams data from its sensors to Astarte, then use this web page to retrieve an overview of the
+device sensors from Astarte.
+
+## Prerequisites
+
+For this example you should:
+
+1. have an Astarte instance up and running with a realm you have access to
+2. install the [example interfaces](../../standard-interfaces/) in the realm
+3. register a device that will use those interfaces in the realm; be sure to note down its device ID
+   and Credentials Secret
+4. modify the `configuration` variable in `index.html` and specify: the `endpoint` of the Astarte
+   instance, the `realm` you chose, a valid JWT `token` that can be used to communicate with
+   AppEngine, the `device` ID your registered.
+
+## Docker walkthrough
+
+You can use Astarte's `stream-qt5-test` to emulate an Astarte device and generate a `datastream`:
+
+```sh
+docker run --net="host" -e "DEVICE_ID=<device_id>" -e "PAIRING_HOST=<pairing_host>" -e "REALM=<realm>" -e "AGENT_KEY=<agent_key>" -e "IGNORE_SSL_ERRORS=true" astarte/astarte-stream-qt5-test:0.11-snapshot
+```
+
+where `device_id` is the device you registered, `pairing_host` is the Astarte endpoint for the
+Pairing APIs, `realm` is the name of your realm, `agent_key` is the valid JWT token of an Agent.
+
+If all went well, `stream-qt5-test` should now be streaming data to Astarte.
+
+Going back to the web page of this example, you can input the device ID to request an overview of
+the device sensors and their data.
+
+## Elixir walkthrough
```

Comment:
While I'm ok with stream-qt5-test as it's a one-liner, i don't think this is the right place for explaining how to use the elixir sdk. I would just mention "... or use an Astarte SDK for the language you're most comfortable with"

Reply:
That's a good point. I was not sure about this.
Showing a use case with one SDK could be beneficial since you have more flexibility and can use more interfaces to send both Values and advertise AvailableSensors, while the docker command can only send Values.
What's your take?

Reply:
Since we can document the use case with `stream-qt5-test`, I removed the walkthrough with the Elixir SDK